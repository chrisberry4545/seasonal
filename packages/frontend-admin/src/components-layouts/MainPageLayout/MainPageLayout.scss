.c-main-page-layout {
  max-width: 100%;
  width: 600px;
  margin: auto;
  padding: 2.5rem 1.25rem;
}
