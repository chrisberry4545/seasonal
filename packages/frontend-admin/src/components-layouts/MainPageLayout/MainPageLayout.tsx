import React, { FC } from 'react';
import './MainPageLayout.scss';

export const MainPageLayout: FC<{}> = ({
  children
}) => (
  <div className='c-main-page-layout'>
    { children }
  </div>
);
