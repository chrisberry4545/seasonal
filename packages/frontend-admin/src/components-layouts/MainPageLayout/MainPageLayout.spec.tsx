import React from 'react';
import { MainPageLayout } from '../MainPageLayout/MainPageLayout';
import { shallow, ShallowWrapper } from 'enzyme';

describe('<MainPageLayout />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
      <MainPageLayout>
        Children
      </MainPageLayout>
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());

});
