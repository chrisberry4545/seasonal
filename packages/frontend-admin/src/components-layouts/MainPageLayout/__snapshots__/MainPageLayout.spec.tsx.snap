// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainPageLayout /> renders correctly 1`] = `
<div
  className="c-main-page-layout"
>
  Children
</div>
`;
